runelite-client/src/main/java/net/runelite/client/plugins/mta/MTAConfig.java

```diff
@@ -27,50 +27,105 @@
 import net.runelite.client.config.Config;
 import net.runelite.client.config.ConfigGroup;
 import net.runelite.client.config.ConfigItem;
+import net.runelite.client.config.ConfigSection;
 
-@ConfigGroup("mta")
+@ConfigGroup(MTAConfig.GROUP)
 public interface MTAConfig extends Config
 {
+	String GROUP = "mta";
```

Comment:
Since this is never referenced elsewhere, this can be reverted.

Reply:
Reverted.